### PR TITLE
Test libcamgm via yast2-ca-management in x11

### PR DESCRIPTION
**Pull request by dehai**
This case is based on below test case:
https://bugzilla.suse.com/tr_show_case.cgi?case_id=1595188
and it can be run successful in non-fips mode, failed in fips mode with exist bug:
https://bugzilla.suse.com/show_bug.cgi?id=1033977
non-fips veriy run:http://10.67.18.165/tests/161
fips failed run:http://10.67.18.165/tests/160

submit this automated case, in case of verified this case in fips and non-fips mode
just can push solve the exist bug with fips mode

### CODE REVIEW

**Comment by dehai** [comment]
related MR:https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/542

File: tests/x11/libcamgm.pm [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+my $email    = "you\@example.com";
+
+sub restart_yast2_camgm {
+    assert_and_click("xterm");
+    type_string("yast2 ca_mgm\n");
+    wait_still_screen 1;
+    assert_screen("yast2-ca-TestCA");
+    send_key("alt-t");
+    wait_still_screen 1;
+    type_string_slow("$password");
```
Comment (okurz) [inline tests/x11/libcamgm.pm]: Why do you need to type slow?

Comment (dehai) [reply tests/x11/libcamgm.pm]: have modified without slow

**Comment by dehai** [comment]
local verified:http://10.67.18.165/tests/164

**Comment by okurz** [comment]
@asmorodskyi please stop doing that! Don't merge new tests when they need needles which are not merged yet - for a good reason

**Pull request merged**